src/python/visclaw/data.py

```diff
@@ -621,12 +622,17 @@ def __init__(self, name, figno, fig_type, plotdata):
         self._show = True
         self.add_attribute('plotaxes_dict', {})
         self.add_attribute('type',fig_type)   # = 'each_frame' or 'each_run' or 'each_gauge'
+        self.add_attribute('use_for_kml',False)
+        self.add_attribute('kml_dpi',200)
+        self.add_attribute('kml_xlimits',None)
```

Comment:
I agree that maybe "kml_xlimits" and "`kml_ylimits" are not the best choice of names - it could be confused with the plotaxes attributes.  

As for the name 'kml_figure',  the user can of course name this figure anything they want.  But I did have in mind that a user would use a figure solely for GE plots and not just switch on or off the use_for_kml setting.  That said, setting "use_for_kml=False" should restore a plot to normal settings.

Also, if a figure is to used for GE, then titles, colorbars, tick marks, labels are ignored.   In fact, there are only very few things that the user can really set for a GE plot, so I didn't think it wouldn't be too much extra code to have a dedicated GE figure (hence the name "kml_figure").   That said, the GE figures all show up in the usual HTML output.  I am not sure this is so helpful, other than to verify that the figures look okay.